### ISSUE TITLE
About SFENCE.W.INVAL & SFENCE.INVAL.IR in U mode

### ISSUE THREAD

**Issue by xiaochunsong92** [riscv/virtual-memory#8]
When execute SFENCE.W.INVAL or SFENCE.INVAL.IR in U mode, Does it raise an illegal instruction exception?

**Comment by huxuan0307** [comment]
> When execute SFENCE.W.INVAL or SFENCE.INVAL.IR in U mode, Does it raise an illegal instruction exception?

Yes. You can find the description of illegal instruction exception for this situation in privileged spec
